sql/geovisualizer.sql

```diff
@@ -23,3 +23,11 @@ CREATE TABLE IF NOT EXISTS states
     country CHARACTER VARYING(165) DEFAULT NULL::CHARACTER VARYING,
     boundary public.geometry(Geometry,4326)
 );
+
+CREATE TABLE IF NOT EXISTS users
+(
+    id SERIAL PRIMARY KEY,
+    name CHARACTER VARYING(64) ,
+    email CHARACTER VARYING(165) UNIQUE,
+    password CHARACTER VARYING(64) 
```

Comment:
Should specify that these fields can not be null. This will add a further level of validation on these fields.

I tried signup API using this body:
{
    "email": null,
    "password": "test"
}
and this gave a success response, instead of giving an error response.

Hence apart from API validations as suggested above, we will have DB level constraints too.